style.css

```diff
@@ -1,41 +1,45 @@
 body {
-    background-color: #7FFFD4;
-    max-width: 800px;
+    background-color: #E0FFFF;
     margin: 0 auto;
+    text-align: center;
 }    
 h1 {
-    font-family: Georgia, cursive;
-    color: #800000;
+    font-family: Georgia;
+    color: #0000CD;
     margin: 1em;
     text-align: center;
-    
     height: 50px;
+    font-size: 2.2em;
```

Comment:
It's very unusual to use em for font-size. Do we need to modify the H1 font-size altogether?